aws_iam_tester/__init__.py

```diff
@@ -1,9 +1,6 @@
-import toml
-from pathlib import Path
+from importlib.metadata import version
 
-def get_version():
-   path = Path(__file__).resolve().parents[1] / 'pyproject.toml'
-   pyproject = toml.loads(open(str(path)).read())
-   return pyproject['tool']['poetry']['version']
-
-__version__ = get_version()
\ No newline at end of file
+try:
+   __version__ = version(__name__)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```